qa/SidechainTestFramework/account/eoa_util.py

```diff
@@ -130,5 +130,9 @@ def eoa_transaction(node, *,
                                               max_priority_fee_per_gas=max_priority_fee_per_gas, chain_id=chain_id)
 
     response = node.rpc_eth_signTransaction(payload)
-    response = node.rpc_eth_sendRawTransaction(response['result'])
-    return response["result"]
+
+    if "result" in response:
+        response = node.rpc_eth_sendRawTransaction(response['result'])
+        return response["result"]
```

Comment:
Missing one last check for `"result" in response` after `rpc_eth_sendRawTransaction`.